Source/WebCore/dom/ViewTransitionTypeSet.h

```diff
@@ -55,7 +55,7 @@ class ViewTransitionTypeSet : public RefCounted<ViewTransitionTypeSet> {
     ViewTransitionTypeSet(Document&, Vector<AtomString>&&);
 
     ListHashSet<AtomString> m_typeSet;
-    Document& m_document;
+    CheckedPtr<Document> m_document;
```

Comment:
m_document cannot be null so this should be a CheckedRef, not a CheckedPtr.

Reply:
Right, not sure why I went from a reference to a pointer there.